packages/eas-cli/src/commands/workflow/run.ts

```diff
@@ -0,0 +1,99 @@
+import chalk from 'chalk';
+import fs from 'node:fs';
+import * as path from 'node:path';
+
+import { getWorkflowRunUrl } from '../../build/utils/url';
+import EasCommand from '../../commandUtils/EasCommand';
+import { WorkflowProjectSourceType } from '../../graphql/generated';
+import { WorkflowRunMutation } from '../../graphql/mutations/WorkflowRunMutation';
+import Log, { link } from '../../log';
+import { getOwnerAccountForProjectIdAsync } from '../../project/projectUtils';
+import { uploadAccountScopedEasJsonAsync } from '../../project/uploadAccountScopedEasJsonAsync';
+import { uploadAccountScopedProjectSourceAsync } from '../../project/uploadAccountScopedProjectSourceAsync';
+
+export default class WorkflowRun extends EasCommand {
+  static override description = 'Run an EAS workflow';
+  static override aliases = ['workflow:run'];
```

Comment:
why is it needed? I believe if a command is in the `commands/workflow` directory and is put in `run.ts` file it will automatically be `eas workflow:run` 🤔

Reply:
Removed.